.travis.yml

```
@@ -2,9 +2,9 @@ sudo: true
 
 jobs:
   include:
-    - name: Sdk tests
-      stage: Tests
-      sudo: false
+    - stage: Tests
+      name: "Unit Tests"
+      if: type = pull_request OR (type = push && branch =~ /^master|[0-9]+-(dev|stable)$/) OR type = cron
```

nitpincking: unneeded parenthesis and `||`/`&&` mix with `AND`/`OR`
```suggestion
      if: type = pull_request OR type = push AND branch =~ /^master|[0-9]+-(dev|stable)$/ OR type = cron
```